v2/cli/init_command.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
+	Language string
+	Git      string
+}
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a template Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to create " +
+				"(required)",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:    flagLanguage,
+			Aliases: []string{"l"},
+			Usage:   "Set the language of the brigade project file to either JavaScript or TypeScript (default)",
+		},
+		&cli.StringFlag{
+			Name:    flagGit,
+			Aliases: []string{"g"},
+			Usage:   "Initialize the brigade project from an existing git repo ",
+		},
+	},
+	Action: initialize,
+}
+
+// Template project.yaml file for a non-git project
+var newProjectFileBytes = []byte(
+	`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+	id: {{ .ID }}
+description: Your newly-initialized Brigade project!
+spec:
+	workerTemplate:
+		logLevel: DEBUG
+		defaultConfigFiles:
+			brigade.{{ .Language }}: |
+				console.log("Hello, World!");
+`)
+
+// Template project.yaml file for a user who provided a git repo link
+var newGitProjectFileBytes = []byte(
+	`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+	id: {{ .ID }}
+description: Your newly-initialized Brigade project!
+spec:
+	workerTemplate:
+		logLevel: DEBUG
+		git:
+			cloneURL: {{ .Git }}
+`)
+
+// Handle init CLI flags
+func initialize(c *cli.Context) error {
+	filename := ".brigade/project.yaml"
+	// allowedLanguages := []string{"js", "ts"}
+
+	projectID := c.String(flagID)
+	language := strings.ToLower(c.String(flagLanguage))
+	gitRepo := c.String(flagGit)
+
+	// Check if inputted language is valid
+	if language == "" {
+		language = "ts"
+	} else if language != "js" && language != "ts" {
+		fmt.Println("Language flag must be set to either js (JavaScript) or ts (TypeScript)")
+		return nil
+	}
```

Comment:
Suggest rewriting as follows:

```
switch (string.ToLower(language)) {
  case: "ts", "typescript":
    // whatever
  case: "js", "javascript":
    // whatever
  default:
    return errors.Errorf("Unrecognized value %q for --language flag", language)
}
```
The most important things here are:

* Not case sensitive
* Easier to amend when we add support for other languages (2.1)
* Error is returned and existing global error handling takes care of it